layouts/partials/header.html

```diff
@@ -14,14 +14,19 @@
           />
           <path d="M0 0h170v76H0z" fill="none" />
         </svg>
-        <span>Sonne tanken</span>
+        <span>{{i18n "sonne_tanken"}}</span>
       </a>
       <nav class="header__navigation">
-        <a href="https://docs.evcc.io/blog" class="header__navigation__item">Blog</a>
-        <a href="https://docs.evcc.io" class="header__navigation__item hide-lg">Doku</a>
-        <a href="https://docs.evcc.io" class="header__navigation__item hide-sm">Dokumentation</a>
+        <a href="https://docs.evcc.io/blog" class="header__navigation__item">{{ i18n "blog" }}</a>
+        <a href="https://docs.evcc.io" class="header__navigation__item hide-lg">{{ i18n "dokumentation-short" }}</a>
+        <a href="https://docs.evcc.io" class="header__navigation__item hide-sm">{{ i18n "dokumentation" }}</a>
         <a href="https://github.com/evcc-io/evcc" class="header__navigation__item">GitHub</a>
-      </nav>
+        {{ range $.Site.Home.AllTranslations }}
```

Comment:
Absolutely not happy with this, but it works to get this out of the door. Would greatly appreciate someone else improving with a picker of some description.

Reply:
I've introduced a simple `DE | EN` lang switcher and adjusted the CSS to make this work on mobile.